packages/instui-cli/lib/commands/create.js

```diff
@@ -63,35 +64,50 @@ exports.builder = (yargs) => {
   )
 
   const packageCommand = 'package'
+  const packageFormatInstructions = ' (in kebab-case, e.g. my-package)'
   yargs.command(
     packageCommand,
-    'Create an Instructure UI package. DEPRECATED, this will be deleted in InstUI 9',
+    'Create an Instructure UI package. FOR INTERNAL USE ONLY!',
     (yargs) => {
       const options = { yargs, contentType: packageCommand }
       generatePathOption(options)
-      generateNameOption(options)
+      generateNameOption({
+        ...options,
+        formatInstructions: packageFormatInstructions
+      })
     },
     async (argv) => {
       const { name, path } = argv
-      handleCreate({ contentType: packageCommand, path, name })
+      handleCreate({
+        contentType: packageCommand,
+        path,
+        name,
+        formatInstructions: packageFormatInstructions
+      })
     }
   )
 
   const componentCommand = 'component'
+  const componentFormatInstructions = ' (in PascalCase, e.g. NumberInput)'
   yargs.command(
     componentCommand,
-    'Create an Instructure UI component. DEPRECATED, this will be deleted in InstUI 9',
+    'Create an Instructure UI component. FOR INTERNAL USE ONLY!',
```

Comment:
I am not sure if we truly want to have this internally only. I think anyone could and should use this, just don't assume that it won't have breaking changes

Reply:
We agreed on the standup that we should leave it "internal use only" for now, because it would require a lot more of setup, documentation and guides to make it easy-to-use for other teams too.